src/crawlee/http_clients/curl_impersonate.py

```diff
@@ -194,7 +195,7 @@ def _get_client(self, proxy_url: str | None) -> AsyncSession:
             # are set as default options.
             kwargs: dict[str, Any] = {
                 'proxy': proxy_url,
-                'impersonate': BrowserType.chrome,
+                'impersonate': BrowserType.chrome124,
```

Comment:
Isn't there something like a "newest chrome" option?

Reply:
Unfortunately, no. The `BrowserType.chrome` was the most recent option, but it has been removed. However, I discovered that there is a `DEFAULT_CHROME` constant that represents the latest version, so we can use that instead.